src/Backend/Modules/Pages/Domain/ModuleExtras/ModuleExtra.php

```diff
@@ -134,6 +134,15 @@ public function getData(): ?string
         return $this->data;
     }
 
+    public function unserializedData(): ?string
```

Comment:
wouldn't it be better to use a LifeCycleCallback here and automatically serialise and unserialise it?
Check out the Meta entity for an example https://github.com/forkcms/forkcms/blob/master/src/Common/Doctrine/Entity/Meta.php#L174

Reply:
I wasn't aware of this technique. Changed in 01db59a0